strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/maven/MavenArtifactControllerTest.java

```diff
@@ -472,7 +473,31 @@ public void testDeleteArtifactDirectory()
         assertFalse("Failed to delete artifact file '" + deletedArtifact.getAbsolutePath() + "'!",
                     deletedArtifact.exists());
     }
-
+    
+    @Test
+    public void testNonExistingDirectoryDownload()
+    {
+        String path = "/storages/storage-common-proxies/maven-central/john/doe/";
```

Comment:
`String path = "/storages/storage-common-proxies/maven-central/john/doe/";`

Could you please write one more test for:

`String path = "/storages/storage-common-proxies/maven-central/john/doe";` ? Notice removed trailing slash. Thanks